### ISSUE TITLE
TextBuffer.on is deprecated.

### ISSUE THREAD

**Issue by Schmidtwar** [Glavin001/atom-beautify#302]
Use TextBuffer::onWillSave instead. A TextBuffer instance is no longer provided as a callback argument.
```
TextBuffer.on (C:\Users\Schmidtwar\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:1179:16)
Object.subscribeWith (C:\Users\Schmidtwar\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\emissary\lib\subscriber.js:28:32)
Object.subscribe (C:\Users\Schmidtwar\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\emissary\lib\subscriber.js:63:21)
<unknown> (C:\Users\Schmidtwar\.atom\packages\atom-beautify\src\beautify.coffee:322:13)
```

**Comment by Schmidtwar** [comment]
When I'm saving this error comes up.

**Comment by Glavin001** [comment]
Duplicate of #289.

**Comment by Glavin001** [comment]
Published to v0.26.5
